build.sbt

```diff
@@ -41,9 +41,11 @@ lazy val Lift = project.dependsOn(Tape, ProjectRef(file("RAII.scala"), "asynchro
 
 lazy val math = project.dependsOn(Lift)
 
-lazy val TapeTask = project.dependsOn(Tape, ProjectRef(file("RAII.scala"), "asynchronous"))
+lazy val `differentiable-Any` = project.dependsOn(Tape, ProjectRef(file("RAII.scala"), "asynchronous"))
 
-lazy val LogRecords = project.dependsOn(Caller)
+lazy val logs = project.dependsOn(Caller)
+
+lazy val `differentiable` = project.dependsOn(`differentiable-Float`, `differentiable-INDArray`)
```

Comment:
Could you explicitly add the `dependsOn` to  `differentiable-Any` and  `differentiable-Double` here?